src/Coin.cpp

```diff
@@ -388,7 +388,7 @@ std::string TW::deriveAddress(TWCoinType coin, const PublicKey& publicKey) {
     switch (coin) {
     case TWCoinTypeBinance:
         return Tendermint::Address(HRP_BINANCE_TEST, publicKey).string();
-
+    
```

Comment:
we need to revert this

Reply:
> we need to revert this

did you mean deleting the empty line I added?

Reply:
I think he means deleting the spaces.

Reply:
done in 06e0d0d3b5011833279af64ed0118dbcf243567c